Incremental Scripts/5_midi_and_notes_to_psychopy.py

```diff
@@ -0,0 +1,110 @@
+# This script will port the previous script number 4 into psychopy.
+# psychopy functions
+from psychopy.gui import DlgFromDict
+from psychopy.visual import Window, TextStim, ImageStim
+from psychopy.core import wait, Clock, quit
+from psychopy.hardware.keyboard import Keyboard
+from pandas import read_csv
+
+# midi functions
+from pygame import midi
+
+
+#exp_info = {'participant_nr': ''}  # no default!
+#dlg = DlgFromDict(exp_info)
+
+win = Window(fullscr=True,size= (2560,1440))
+win.units = 'pix'
+
+# initialise clock, key and midi input from psychopy and pygame input 
+
+clock = Clock()
+
+kb = Keyboard()
+
+welcome_txt_stim = TextStim(win,
+ text="""Welcome to this experiment!
+ You will be presented with piano notes and have to press the corresponding key on the piano. 
+ Afterwards you will receive feedback
+ (Press enter to start)"""
+)
+
+welcome_txt_stim.draw()
+
+win.flip()
+
+while True:
+    keys = kb.getKeys()
+    if "return" in keys:
+        break
+
+
+# Now I will create two dataframes: The first one 'key_bindings' has the note files and the corresponding key_id that we implemented earlier
+# The second "condition_df" dataframe will include the order in which the notes will be shown (random for now) and store the reaction time
+
+key_bindings = read_csv("./key_bindings.csv",index_col=0)
+
+
+#condition_df = key_bindings.sample(frac=1)
+condition_df = key_bindings
+midi.init()
+
+input = midi.Input(1)
+
```

Comment:
Maybe some comments/documentation about the code from line 53 to 90 would be helpful to understand what is going on. But maybe I´m just too slow to fully understand it :D